relayer/src/chain/cosmos.rs

```diff
@@ -1516,13 +1544,20 @@ fn do_health_check(chain: &CosmosSdkChain) -> Result<(), Error> {
         return Err(Error::no_historical_entries(chain_id.clone()));
     }
 
-    let status = chain.status()?;
+    let status = chain.chain_status()?;
 
     // Check that transaction indexing is enabled
     if status.node_info.other.tx_index != TxIndexStatus::On {
         return Err(Error::tx_indexing_disabled(chain_id.clone()));
     }
 
+    // Check that the chain identifier matches the network name
```

Comment:
Note: This query has not much to do with the present PR. It's just an additional non-critical health check that can reveal misconfigurations of config.toml.